tukaan/_wm.py

```diff
@@ -45,71 +45,73 @@ class WindowCompositionAttributeData(ctypes.Structure):
 
 
 class WMProperties:
-    def _get_title(self) -> str:
+    @property
+    def title(self) -> str:
         return Tcl.call(str, "wm", "title", self._wm_path)
 
-    def _set_title(self, new_title: str) -> None:
-        Tcl.call(None, "wm", "title", self._wm_path, new_title)
-
-    title = property(_get_title, _set_title)
+    @title.setter
+    def title(self, value: str):
+        Tcl.call(None, "wm", "title", self._wm_path, value)
 
+    @property
     @Tcl.update_before
-    def _get_x(self) -> int:
+    def x(self) -> int:
         return Tcl.call(int, "winfo", "x", self._wm_path)
 
+    @x.setter
     @Tcl.update_after
-    def _set_x(self, new_x: int) -> None:
-        Tcl.call(None, "wm", "geometry", self._wm_path, f"+{new_x}+{self._get_y()}")
-
-    x = property(_get_x, _set_x)
+    def x(self, value: int) -> None:
+        Tcl.call(None, "wm", "geometry", self._wm_path, f"+{value}+{self.y}")
 
+    @property
     @Tcl.update_before
-    def _get_y(self) -> int:
+    def y(self) -> int:
         return Tcl.call(int, "winfo", "y", self._wm_path)
 
+    @y.setter
     @Tcl.update_after
-    def _set_y(self, new_y: int) -> None:
-        Tcl.call(None, "wm", "geometry", self._wm_path, f"+{self._get_x()}+{new_y}")
+    def y(self, value: int) -> None:
+        Tcl.call(None, "wm", "geometry", self._wm_path, f"+{self.x)}+{value}")
```

Comment:
```suggestion
        Tcl.call(None, "wm", "geometry", self._wm_path, f"+{self.x}+{value}")
```

Reply:
Uhh typo, hard to find fr cuz I have enabled strict pyright analysis and it finds tons of errors.